doc/pages/services.json

```diff
@@ -172,5 +172,17 @@
     "link": "https://www.projectcallisto.org/",
     "tags": ["sexual_assault_survivors", "harassment", "reporting"],
     "languages": ["en"]
+  },
+  {
+    "name": "Headspace",
```

Comment:
We actually already have this on our page along with other meditation apps that are great.